RubeesEat/Model/Bill.cs

```diff
@@ -2,7 +2,7 @@
 
 namespace RubeesEat.Model;
 
-public class Bill
+public class Bill : IComparable<Bill>
```

Comment:
Please revert the comparability here. See the other comment on how we can sort without it.